udata/tests/forms/test_tag_field.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
As I prefer to have tags handling in a seaparate module, I prefer to have tests in `udata/tests/test_tags.py`.